roles/etcd/tasks/main.yml

```diff
@@ -27,3 +27,11 @@
     daemon_reload: yes
     enabled: yes
     state: started
+
+- name: Renewal etcd certificates
+  import_tasks: renewal.yml
+  when: upgrade
+
+- name: Distribute etcd certificates from etcd to control plane nodes
+  import_tasks: distribute.yml
+  when: not etcd_on_control_plane | bool
```

Comment:
why is the casting to bool needed?

Wouldn't it be better to have it as one of the defaults of the role set it to `false`?